src/main/scala/is/hail/methods/SampleQC.scala

```diff
@@ -40,6 +19,7 @@ object SampleQCCombiner {
     "nSingleton" -> TInt32,
     "nTransition" -> TInt32,
     "nTransversion" -> TInt32,
+    "nStar" -> TInt32,
```

Comment:
Should the type of these be TInt64? Max int is less than the number of base pairs in the genome.